config/workers.coffee

```diff
@@ -318,7 +318,7 @@ module.exports = (KONFIG, options, credentials) ->
         ]
 
     userproxies         :
-      group             : 'proxy'
+      group             : if KONFIG.environment is 'default' then 'default' else 'proxy'
```

Comment:
Hm i forgot about tunnel workers - lets have this change then we can take a look later